ext/AvizExt/AvizExt.jl

```diff
@@ -511,21 +548,25 @@ function ADRIA.viz.explore(rs::ResultSet)
         if @isdefined up_timer
             close(up_timer)
         end
-        up_timer = Timer(x -> update_disp(time_val, tac_val, rcp45, rcp60, rcp85, c_tog, u_tog, g_tog, intervs...), 2)
+        up_timer = Timer(
+            x -> update_disp(
+                time_val, tac_val, rcp45, rcp60, rcp85, c_tog, u_tog, g_tog, intervs...
+            ),
+            2
+        )
     end
 
     @info "Displaying UI"
     gl_screen = display(f)
     # DataInspector()
 
-    wait(gl_screen)
+    return wait(gl_screen)
```

Comment:
would you prefer this instead?
```julia
wait(gl_screen)
return
```

Reply:
Explicit return is better imo.

And yes, what you suggest is fine with me

Reply:
I've added `return nothing` in appropriate places. I think I've caught them all, but it won't hurt anything if I missed one since the formatter is preserving current behavior.